mimic/model/mailgun_objects.py

```diff
@@ -118,5 +118,5 @@ def filter_message_by_to_address(self, to_address):
         Retrieve a :obj:`Message` object by its `to` address.
         """
         for each_msg in self.message_store:
-            if each_msg.to == to_address[0]:
+            if each_msg.to == to_address:
```

Comment:
Previously `to_address` was just being pulled straight out of `request.args`, which left the signature of this method somewhat nonsensical; `to_address` was a `list` of `bytes`.  Now it's actually a `text_type`, which makes more sense for it to be called `to_address` (i.e. not plural) and to be compared against `each_msg.to`, which is also a `text_type`.  There's only one caller and it's been adjusted to do the work of extracting the argument from `request.args` fully (pulling it out of the list, decoding it).